languages/common-lisp/concepts/conditionals/links.json

```diff
@@ -1 +1,10 @@
-[]
+[
+  {
+    "url": "https://riptutorial.com/common-lisp/example/11082/conditional-constructs",
+    "description": "A tutorial"
```

Comment:
Maybe it would be good to give slightly more detail here, even if it's just "A tutorial about the different conditional constructs"